src/pages/Course.tsx

```diff
@@ -32,7 +32,7 @@ export default function Setup({
           name="students"
           value={input.students}
           onChange={handleListChange}
-        ></Textarea>
+        />
```

Comment:
I thought self closing textareas produce a react warning, but apparently they do not. Good catch!